src/Data/Nat/Properties.agda

```diff
@@ -657,6 +657,17 @@ i*j≡1⇒j≡1 i j eq = i*j≡1⇒i≡1 j i (trans (*-comm j i) eq)
   ; ε-homo  = refl
   }
 
+^-assoc-^-* : ∀ m n p → (m ^ n) ^ p ≡ m ^ (n * p)
+^-assoc-^-* m n zero    = begin
+  1           ≡⟨⟩
+  m ^ 0       ≡⟨ cong (m ^_) (*-comm 0 n) ⟩
```

Comment:
Also, very, very minor, but this should probably be `*-identity` rather than `*-comm`.

Reply:
Sorry I meant `*-zero`?

Reply:
Ha! Makes sense. I'll do that then. Thanks.